internal/test/t1/t1.bnf

```diff
@@ -1,10 +1,16 @@
 !whitespace : ' ' ;
 
 A 
-	: B "c" << []interface{}{$0, "c"}, nil >>
+	: B "a" << []interface{}{$0, "a"}, nil >>
+	| C "a" << []interface{}{$0, "a"}, nil >>
 	;
 
 B
-	: empty 
+	: empty
 	| "b"		<< "b", nil >>
+	;
+
+C
+	:
```

Comment:
I thought that gocc should give an error instead of generating the code for this example?
So it is probably easier to make a separate test.